src/chatlog/chatlog.cpp

```diff
@@ -456,11 +456,18 @@ void ChatLog::startResizeWorker()
             workerAnchorLine = visibleLines.first();
     }
 
+    // switch to busy scene displaying the busy notification if there is a lot
+    // of text to be resized
+    int txt = 0;
+    for(ChatLine::Ptr line : lines)
```

Comment:
Yeah quitting the loop is right, saves a lot of computations. So i have amended the commit.
Also i experimented with this code
```
    long long t = QDateTime::currentMSecsSinceEpoch();
    for (int i=0; i<500000; i++);
    qDebug()<<QDateTime::currentMSecsSinceEpoch()-t;
```
Sadly i got 1 or 2 ms for <code>i \< 500000</code>. And 15 to 17 ms for <code>i \< 500000</code>. 

Specs - i7 4702MQ, 8 gb ram and my qTox runs on ubuntu 14.04.